src/Jackett.Common/Definitions/audiences.yml

```diff
@@ -163,5 +163,5 @@ search:
       text: 172800
     description:
       selector: td:nth-child(2)
-      remove: a, img
+      remove: a, b, font, img, span
```

Comment:
this reduces 
![image](https://user-images.githubusercontent.com/6933240/236599007-61ffec8d-100a-4d92-986b-0a5e8cd6cfee.png)
to just
![image](https://user-images.githubusercontent.com/6933240/236599196-7eb91114-8db4-4e3c-9f67-0f3f0b919acb.png)

Reply:
wtf where are my pictures?